cluster-provision/k8s/scripts/node01.sh

```diff
@@ -10,14 +10,20 @@ done
 
 kubeadm init --config /etc/kubernetes/kubeadm.conf
```

Comment:
maybe we should add `--pod-network-cidr=$pod_cidr` here as well?

Reply:
When invoking 'kubeadm init' you cannot combine --config with any other parameter.
I verified and it works like that.